src/components/Mapa.jsx

```diff
@@ -1,185 +1,176 @@
-
```

Comment:
Mapa component was already on the project, it was called Caba.jsx and I renamed it Mapa because it was confusing. I'm trying to design map styles, and markers but it's being difficult without them. Maybe we can connect the project to a provisional google sheet so i can design markers.